bin/ember-template-lint.js

```diff
@@ -245,16 +249,30 @@ async function run() {
     process.exitCode = 1;
   }
 
+  printResults(results, { options, todoInfo });
+}
+
+function printResults(results, { options, todoInfo }) {
   let hasErrors = results.errorCount > 0;
   let hasWarnings = results.warningCount > 0;
   let hasTodos = options.includeTodo && results.todoCount;
   let hasUpdatedTodos = options.updateTodo;
 
-  let printer = new Printer({
+  let formatter = loadFormatter({
     ...options,
     hasResultData: hasErrors || hasWarnings || hasTodos || hasUpdatedTodos,
   });
-  printer.print(results, todoInfo);
+
+  let output = formatter.format(results, todoInfo);
```

Comment:
Let's check if `typeof formatter.format === 'function'` and if not fallback to existing behavior (makign this non-breaking)

Reply:
100%. This would be easy to switch back to using `print`. Will make that change.